app/services/sla_breach_service.rb

```diff
@@ -10,7 +10,12 @@ def breach_threshold
 
   # attention_needed_since is present and is before breach threshold
   # returns data in the format { vita_partner_id: breach_count }
-  def attention_needed_breach
+  def attention_needed_breaches
     Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:attention_needed_since].lteq(breach_threshold)).count(:id)
   end
+
+  # clients who messaged us and have not been responded to _with a message, call or email_ within the breach threshold
+  def outgoing_communication_breaches
```

Comment:
This PR adds the query, but a subsequent one will add this data to the metrics view -- I want to make sure we have the seeded data in prod before we add the view for it.

Reply:
Awesome!